crates/isograph_schema/src/refetch_strategy.rs

```diff
@@ -25,7 +25,7 @@ pub enum RefetchStrategy<
             TSelectionTypeSelectionLinkedFieldAssociatedData,
         >,
     ),
-    // RefetchFromRoot
+    RefetchFromRoot(RootOperationName),
```

Comment:
we never use the `RootOperationName`. So I think this is broken for refetching things on `Mutation`. Can we add two test cases to `pet-demo`, where we refetch something that's on query and on mutation?

(If I have time... probably tomorrow, I can modify the pet demo to fetch from Mutation etc etc)